loggingservice/logging.yaml

```diff
@@ -2,7 +2,7 @@ openapi: 3.0.0
 info:
   title: Rokwire Logging Building Block API
   description: Logging Building Block API Documentation
-  version: 1.14.1
+  version: 1.15.0
```

Comment:
Please update the version in the YAML file as the next deployment of this module will need the latest version. Thanks.